ilastik/applets/dataExport/dataExportGui.py

```diff
@@ -223,11 +227,86 @@ def initCentralUic(self):
 
         # Set up handlers
         self.batchOutputTableWidget.itemSelectionChanged.connect(self.handleTableSelectionChange)
+        # Handle the right-click event
+        self.batchOutputTableWidget.setContextMenuPolicy(Qt.CustomContextMenu)
+        self.batchOutputTableWidget.customContextMenuRequested.connect(self.showContextMenu)
 
         # Set up the viewer area
         self.initViewerStack()
         self.splitter.setSizes([150, 850])
 
+    # List of all valid file extensions that ilastik might export.
+    VALID_EXPORT_EXTENSIONS = {
+        ".tiff",
+        ".tif",
+        ".h5",
+        ".png",
+        ".jpg",
+        ".jpeg",
+        ".bmp",
+        ".npy",
+        ".csv",
+        ".tsv",
+        ".xlsx",
+        ".hdf5",
+    }
```

Comment:
How did you generate this list? it is not complete and includes values that ilastik does not export to. Also comment wise, please don't add comments where it is super obvious. With a variable name `VALID_EXPORT_EXTENSIONS` it is clear enough.